Source/KeyboardManager.swift

```diff
@@ -5,35 +5,51 @@
 
 import UIKit
 
-public typealias KeyboardManagerEventClosure = (KeyboardManagerEvent, KeyboardManagerEventData) -> Void
+public typealias KeyboardManagerEventClosure = (KeyboardManagerEvent) -> Void
 
 public enum KeyboardManagerEvent {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/alphatroya/KeyboardManager/blob/a51863b5a29b6326270bd93f1c4098a9bafc347f/Source/KeyboardManager.swift#L10"></span>Public declarations should be documented.</td>
    </tr>
  </tbody>
</table>